{"errorMessage": "An error occurred (UnauthorizedOperation) when calling the DescribeInstances operation: You are not authorized to perform this operation. User: arn:aws:sts::084828593269:assumed-role/ec2-monitoring-lambda-role/ec2-monitoring-lambda is not authorized to perform: ec2:DescribeInstances because no identity-based policy allows the ec2:DescribeInstances action", "errorType": "ClientError", "requestId": "bea26500-0e6e-4504-8602-96ebf5f7305c", "stackTrace": ["  File \"/var/task/lambda_monitor.py\", line 88, in lambda_handler\n    scan_and_create_for_all_running()\n", "  File \"/var/task/lambda_monitor.py\", line 73, in scan_and_create_for_all_running\n    reservations = ec2.describe_instances(\n", "  File \"/var/runtime/botocore/client.py\", line 602, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/runtime/botocore/context.py\", line 123, in wrapper\n    return func(*args, **kwargs)\n", "  File \"/var/runtime/botocore/client.py\", line 1078, in _make_api_call\n    raise error_class(parsed_response, operation_name)\n"]}